### PR TITLE
RHIF-34 Create tabs for available and completed tasks

### PR DESCRIPTION
**Pull request by johnsonm325**

### CODE REVIEW

**Review by adonispuente — APPROVED** [review]
Everything works well, just need to delete the commented out import in TasksTabs

File: src/SmartComponents/AvailableTasks/AvailableTasks.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+const AvailableTasks = () => {
+  return (
+    <React.Fragment>
+      <div aria-label="available-tasks">Available tasks</div>
+    </React.Fragment>
+  );
+};
+
+export default withRouter(AvailableTasks);
```
Comment (leSamo) [inline src/SmartComponents/AvailableTasks/AvailableTasks.js:12]: `withRouter` is and older paradigm of react-router-dom which has been removed in latest major version (v6), they instead recommend using Hooks;

also maybe it's just a placeholder, but it does not do anything in the current code

[useHistory](https://v5.reactrouter.com/web/api/Hooks/usehistory)
[useLocation](https://v5.reactrouter.com/web/api/Hooks/uselocation)
[useParams](https://v5.reactrouter.com/web/api/Hooks/useparams)
[useRouteMatch](https://v5.reactrouter.com/web/api/Hooks/useroutematch)

Comment (johnsonm325) [reply src/SmartComponents/AvailableTasks/AvailableTasks.js:12]: Yes, these will be placeholders for now. I may remove them in a future PR.

**Comment by johnsonm325** [comment]
@leSamo The issues are resolved.

I'm not sure I understand the benefit of `map` vs. `for` loop. `for` loops are known to technically be faster, and I honestly still have an easier time understanding a `for` loop much more than `map`. As far as making it inline, I see the benefit of doing it this once, but often apps inline too many lines of code and it become hard to read. Sometimes functions are just easier to parse through.

All that being said, I think your changes are okay for this PR and I'm happy to make them. This is just a discussion point I'd like to start.

**Pull request merged**